code/modules/reagents/chemistry/recipes/medicine.dm

```diff
@@ -233,6 +233,12 @@
 	results = list("regen_jelly" = 2)
 	required_reagents = list("tricordrazine" = 1, "slimejelly" = 1)
 
+/datum/chemical_reaction/jelly_convert
```

Comment:
This seems overly deadly and rather op to heal slimes and to harm non-slimes...

Reply:
????

No? reagents don't react with blood. this is purely so slimejelly can be produced.